src/playground/service.clj

```diff
@@ -7,6 +7,7 @@
    [io.pedestal.http.body-params :as body-params]
    [io.pedestal.http.route :as route]
    [io.pedestal.interceptor.chain :as interceptor-chain]
+   [io.pedestal.http.ring-middlewares :as middlewares]
```

Comment:
Please follow these: https://stuartsierra.com/2015/05/10/clojure-namespace-aliases

In this case, I'd use `ring-middlewares` or `pedestal.http.ring-middlewares`, or no alias at all.

Reply:
I understand and agree. The `middlewares` alias I saw in pedestal samples, where I found the cookie pattern. https://github.com/pedestal/pedestal/blob/a99159ef221c4fb1055d74fece93088890d7d32e/samples/ring-middleware/src/ring_middleware/service.clj#L18